src/Rector/v9/v0/FindByPidsAndAuthorIdRector.php

```diff
@@ -30,7 +30,7 @@ public function getNodeTypes(): array
      */
     public function refactor(Node $node): ?Node
     {
-        if (! $this->isMethodStaticCallOrClassMethodObjectType($node, SysNoteRepository::class)) {
+        if (! $this->nodeTypeResolver->isMethodStaticCallOrClassMethodObjectType($node, SysNoteRepository::class)) {
             return null;
         }
         if (! $this->isName($node->name, 'findByPidsAndAuthor')) {
```

Comment:
This is a typical example of where `AbstractRector` is heading. There will be less and less global methods accessible everywhere, and more focus on specific services that handles the call.

There method will be probably kept even in the future, as they're very practical for every rule:

- `getName()`
- `isName()`
- `getStaticType()`
- etc